ci/jenkins/templates/gpu_jenkinsfile.groovy.j2

```diff
@@ -193,7 +193,10 @@ def deploy_docs() {
   }
 }
 
-def deploy() {
```

Comment:
I think this should always just be on-demand, this messes with external state (deploying the docs to github pages) and doesn't take too long anyways